pkg/model/pool.go

```diff
@@ -24,6 +24,10 @@ package model
 // atomic pool and can be abstracted from any storage platform.
 type StoragePoolSpec struct {
 	*BaseModel
+	// The uuid of project
+	// + readOnly
+	ProjectId string `json:"projectId"`
```

Comment:
Should be changed to `TenantId`

Reply:
I will commit a another PR to solve this issue.